test/api/bootstrap.test.js

```diff
@@ -4,26 +4,24 @@ require('./support/aws-mocks');
 const sequelize = require('../../api/models').sequelize;
 
 const models = sequelize.models;
-
+const ActionType = models.ActionType;
+const types = ['add', 'remove', 'update'];
 const cleanDatabase = () => {
   const promises = Object.keys(models).map((name) => {
     const model = models[name];
-    const promise = model.sync()
-      .then(() => model.destroy({ where: {} }));
+    // { force: true } removes soft deleted models and prevents uniqueness validation errors
```

Comment:
remove this line since it is commented out?

Reply:
I guess this is unclear, its literally a comment explaining why `{force: true }` is used.

Reply:
Oh, I gotcha. That was mostly my bad skimming then. Maybe slight change like:

```js
// Using `force: true` removes soft deleted models and prevents uniqueness validation errors